packages/api/src/services/lsp-creator.ts

```diff
@@ -0,0 +1,33 @@
+import { clients } from "@uma/sdk";
```

Comment:
this file just uses the lsp-creator client and watches for new deployments, adds them to the registeredLsps set